apps/patient/src/components/StatusStepper.tsx

```diff
@@ -50,7 +50,7 @@ export const StatusStepper = ({ status, fulfillmentType, patientAddress }: Props
               const title = t[fulfillmentType][state].status;
               const isDelivery = fulfillmentType === 'COURIER' || fulfillmentType === 'MAIL_ORDER';
               const description = `${t[fulfillmentType][state].description(isMultiRx)}${
-                isDelivery ? patientAddress : ''
+                isDelivery && state === 'READY' ? patientAddress : ''
```

Comment:
What is this?

Reply:
ah a fix on the stepper, was appending address on each step instead of just READY